settings.py

```diff
@@ -23,13 +21,13 @@
 N_ROUNDS = 1
 CRITERIA = "soft"
 
-SCOPE = [
-    "https://spreadsheets.google.com/feeds",
-    "https://www.googleapis.com/auth/drive",
-]
+# SCOPE = [
+#     "https://spreadsheets.google.com/feeds",
+#     "https://www.googleapis.com/auth/drive",
+# ]
 
 
-SPREADSHEET = "2019-12 DMIA Hat (Responses)"
+# SPREADSHEET = "2019-12 DMIA Hat (Responses)"
```

Comment:
It appears that we don't use these constants anywhere, right? I mean, `SPREADSHEET` and `SCOPE`.

Reply:
Right, we don't use them, so first I commented it to show that it is not used. I will delete it then.